README.md

```diff
@@ -2,9 +2,9 @@
 hiptensor is AMD's C++ library for accelerating tensor primitives based on the composable kernel library, through general purpose kernel languages, like HIP C++.
 
 ## GPU Support
-* AMD CDNA class GPU featuring matrix core support: gfx908, gfx90a as 'gfx9'
+* AMD CDNA class GPU featuring matrix core support: gfx908, gfx90a as 'gfx9', gfx940
```

Comment:
Should read:
AMD CDNA class GPU featuring matrix core support: gfx908, gfx90a, gfx940 as 'gfx9'